### PR TITLE
Branch 3.0

### PR DESCRIPTION
**Pull request by cornelinux**
We fixed all open issues for 3.0.1, so we can merge it again into master.

Well, in master we will probably set the version in setup.py to 3.1dev1 then.

### CODE REVIEW

File: Changelog [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+  Fixes:
+  * Fix PUSH token issues:
+    * Add logic checking to setup of PUSH token (#1592)
+    * Remove double enrollment notification of PUSH token in WebUI (#1598)
+    * Fix to allow spaces in Firebase configuration (#1599)
+    * Add support for iOS Firebase configuration (#1608)
+    * Fix to allow PUSH token enrollment, even with Label-policy (#1589)
+    * Fix to mark PUSH token challenge answered in the database (#1584)
+  * Fix the validity period of the registration token (#1587)
+  * Beautify the vertial alignment in the Web UI top menu (#1559)
```
Comment (fredreichbier) [inline Changelog]: Typo: *vertical

File: doc/conf.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 # built documents.
 #
 # The short X.Y version.
-version = '3.0'
+version = '3.0.1dev1'
```
Comment (fredreichbier) [inline doc/conf.py]: Should this also be ``3.0.1dev2``, like in ``setup.py``?

File: Changelog (hunk 1 above)
Comment (cornelinux) [reply Changelog]: Will do

File: doc/conf.py (hunk 2 above)
Comment (cornelinux) [reply doc/conf.py]: Yes. But the most important part is, that set this to 3.0.1 when we are done! ;-)
Will do.

Comment (cornelinux) [reply doc/conf.py]: Hm, lets see if this appears, when i push on the branch-3.0

Comment (cornelinux) [reply doc/conf.py]: done in 656fe468

File: Changelog (hunk 1 above)
Comment (cornelinux) [reply Changelog]: done in 656fe468

**Comment by fredreichbier** [comment]
Travis has run successfully, but github doesn't know. Merging!

**Pull request merged**